app/src/main/java/org/simple/clinic/bloodsugar/history/BloodSugarHistoryScreen.kt

```diff
@@ -3,30 +3,94 @@ package org.simple.clinic.bloodsugar.history
 import android.content.Context
```

Comment:
Reword the commit message? Ex: `Implement UI methods in BloodSugarHistoryScreen`